modules/headline.py

```diff
@@ -2,6 +2,8 @@
 from bs4 import BeautifulSoup
 from textblob.classifiers import NaiveBayesClassifier
```

Comment:
Textblob not necessary.

Reply:
Required in the train_data and the headline_category as of now. Can be removed once NLTK is implemented.

Reply:
I am working on it. learning how to train data using the same classifier with NLTK. Once done I will make a pull request